packages/mobile/src/screens/create-chat-blast-screen/ChatBlastSelectContentScreen.tsx

```diff
@@ -0,0 +1,89 @@
+import { useCallback, useState } from 'react'
+
+import { useField, useFormikContext } from 'formik'
+import { KeyboardAwareFlatList } from 'react-native-keyboard-aware-scroll-view'
+
+import type { TextInputChangeEvent } from '@audius/harmony-native'
+import {
+  Flex,
+  Radio,
+  RadioGroup,
+  Text,
+  TextInput
+} from '@audius/harmony-native'
+import { ScreenContent } from 'app/components/core'
+import { useRoute } from 'app/hooks/useRoute'
+
+import { FormScreen } from '../form-screen'
+
+const renderItem = ({ item }) => {
+  return (
+    <Flex row ph='xl' pv='l' backgroundColor='white' borderBottom='default'>
+      <Radio
+        value={item.label}
+        label={
+          <Text variant='title' size='l' strength='weak'>
+            {item.label}
+          </Text>
+        }
+      />
+    </Flex>
+  )
+}
+
+export const ChatBlastSelectContentScreen = () => {
+  const { params } = useRoute<'ChatBlastSelectContent'>()
+  const { valueName, title, searchLabel, content } = params
+  const [{ value }, , { setValue }] = useField({
+    name: valueName,
+    type: 'select'
+  })
+  const { submitForm } = useFormikContext()
+  const [search, setSearch] = useState('')
+  const filteredOptions = content.filter((option) => {
+    return search
+      ? option.label.toLowerCase().includes(search.toLowerCase())
+      : true
+  })
+  const radioGroupValue = value ?? ''
+
+  const handleClear = useCallback(() => {
+    setValue(undefined)
+  }, [setValue])
+
+  const handleSearchChange = useCallback(
+    (e: TextInputChangeEvent) => {
+      setSearch(e.nativeEvent.text)
+    },
+    [setSearch]
+  )
+
+  return (
+    <FormScreen
+      title={title}
+      clearable
+      onClear={handleClear}
+      onSubmit={submitForm}
+    >
+      <ScreenContent>
+        {/* TODO: 70% bad */}
```

Comment:
the code is reviewing itself 👀

Reply:
without the 70% height, the footer (done + clear buttons) always covers some of the results :(
better fix would be to do some height calculations. made a ticket for this.